### PR TITLE
loaded animation

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

**Comment by asasmith** [comment]
Need to address the positioning of this `:after` element.

Screen Shot 2021-03-24 at 9 29 07 AM
Screen Shot 2021-03-24 at 9 29 18 AM

File: web/src/components/index/indexIntro.js [hunk 1]
```diff
@@ -1,17 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import tw from 'twin.macro';
+import { motion, AnimatePresence } from 'framer-motion';
 
-import Img from 'gatsby-image';
+// import Img from 'gatsby-image';
```
Comment (asasmith) [inline web/src/components/index/indexIntro.js]: Remove completely.

File: web/src/components/index/indexIntro.js [hunk 2]
```diff
@@ -30,19 +32,26 @@ const Heading = styled.h2`
...
 
 export default function IndexIntro({ content }) {
-    const {
-        introHeading,
-        introPlaceholderImage: {
-            asset: { fluid: imageData },
-        },
-        _rawIntroBody,
-    } = content;
+    const [finishedAnimation, setFinishedAnimation] = useState(false);
```
Comment (asasmith) [inline web/src/components/index/indexIntro.js]: Small thing, can we change this to `isFinishedAnimation` or `isAnimationFinished`? I think the `is` helps to communicate what this prop is for.

**Review by asasmith — CHANGES_REQUESTED** [review]
Check the mobile styles for the text content (`Get a complete picture of your data`). I think the font-size is smaller on small sceens.

**Comment by kcbreuil** [comment]
> Need to address the positioning of this `:after` element.
> 
> Screen Shot 2021-03-24 at 9 29 07 AM
> Screen Shot 2021-03-24 at 9 29 18 AM

Yeah I am curious if we can use a graphic instead? I was trying to fix it last night but didnt have much luck since the container its contingent on changes throughout the animation.

**Comment by asasmith** [comment]
> > Need to address the positioning of this `:after` element.
> > Screen Shot 2021-03-24 at 9 29 07 AM
> > Screen Shot 2021-03-24 at 9 29 18 AM
> 
> Yeah I am curious if we can use a graphic instead? I was trying to fix it last night but didnt have much luck since the container its contingent on changes throughout the animation.

sounds like we need to change the container then.

yeah, if you move the `:after` element to the `h2` instead of the ~body paragraph~ wrapping div it should work. Need to add `relative` to the `h2` as well. 

Let me know if you have any questions

**Comment by kcbreuil** [comment]
> > > Need to address the positioning of this `:after` element.
> > > Screen Shot 2021-03-24 at 9 29 07 AM
> > > Screen Shot 2021-03-24 at 9 29 18 AM
> > 
> > 
> > Yeah I am curious if we can use a graphic instead? I was trying to fix it last night but didnt have much luck since the container its contingent on changes throughout the animation.
> 
> sounds like we need to change the container then.
> 
> yeah, if you move the `:after` element to the `h2` instead of the ~body paragraph~ wrapping div it should work. Need to add `relative` to the `h2` as well.
> 
> Let me know if you have any questions

@asasmith I tried that last night and it still moves up and then down again. I think we need to put it on the `P` element actually, since the wireframes shows that it "reappears" with the text 

[
Screen Shot 2021-03-24 at 1 15 52 PM
Screen Shot 2021-03-24 at 1 15 20 PM

](url)

**Comment by asasmith** [comment]
> > > > Need to address the positioning of this `:after` element.
> > > > Screen Shot 2021-03-24 at 9 29 07 AM
> > > > Screen Shot 2021-03-24 at 9 29 18 AM
> > > 
> > > 
> > > Yeah I am curious if we can use a graphic instead? I was trying to fix it last night but didnt have much luck since the container its contingent on changes throughout the animation.
> > 
> > 
> > sounds like we need to change the container then.
> > yeah, if you move the `:after` element to the `h2` instead of the ~body paragraph~ wrapping div it should work. Need to add `relative` to the `h2` as well.
> > Let me know if you have any questions
> 
> @asasmith I tried that last night and it still moves up and then down again. I think we need to put it on the `P` element actually, since the wireframes shows that it "reappears" with the text
> 
> [
> Screen Shot 2021-03-24 at 1 15 52 PM
> Screen Shot 2021-03-24 at 1 15 20 PM
> 
> ](url)

My guess is the correct parent element didn't have `position: relative`.

**Comment by kcbreuil** [comment]
> > > > > Need to address the positioning of this `:after` element.
> > > > > Screen Shot 2021-03-24 at 9 29 07 AM
> > > > > Screen Shot 2021-03-24 at 9 29 18 AM
> > > > 
> > > > 
> > > > Yeah I am curious if we can use a graphic instead? I was trying to fix it last night but didnt have much luck since the container its contingent on changes throughout the animation.
> > > 
> > > 
> > > sounds like we need to change the container then.
> > > yeah, if you move the `:after` element to the `h2` instead of the ~body paragraph~ wrapping div it should work. Need to add `relative` to the `h2` as well.
> > > Let me know if you have any questions
> > 
> > 
> > @asasmith I tried that last night and it still moves up and then down again. I think we need to put it on the `P` element actually, since the wireframes shows that it "reappears" with the text
> > [
> > Screen Shot 2021-03-24 at 1 15 52 PM
> > Screen Shot 2021-03-24 at 1 15 20 PM
> > ](url)
> 
> My guess is the correct parent element didn't have `position: relative`.

it did. i double checked and anytime i put relative the entire thing disappears. I have a fix, pushing up now

**Review by asasmith — APPROVED** [review]

**Pull request merged**